### ISSUE TITLE
 decreased bone marrow adipose tissue amount

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#2958]
This is a request for a new term:
>1. Preferred term label (e.g., asplenia)

decreased bone marrow adipose tissue amount

>2. Synonyms (e.g., absent spleen)

decreased bone marrow adiposity; decreased bone marrow fat; decreased bone marrow fat amount; decreased marrow fat

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

decreased quantity of the bone marrow fat-storing cells/tissue

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:25934999

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal bone marrow adipose tissue amount (MP:0030008)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjwang

**Comment by sbello** [comment]
MP:0030008 - abnormal
MP:0031008 - decreased
MP:0030009 - increased
